frontend/src/__tests__/cypress/cypress/tests/mocked/modelServing/modelServingGlobal.cy.ts

```diff
@@ -469,7 +469,8 @@ describe('Model Serving Global', () => {
     inferenceServiceModalEdit.findSubmitButton().should('be.disabled');
     inferenceServiceModalEdit.findConnectionNameInput().type('Test Name');
     inferenceServiceModalEdit.findConnectionFieldInput('URI').type('/');
-    inferenceServiceModalEdit.findSubmitButton().click().should('be.disabled');
+    inferenceServiceModalEdit.findConnectionFieldInput('URI').blur();
+    inferenceServiceModalEdit.findSubmitButton().should('be.disabled');
```

Comment:
I'm not sure why I hit this? Did I mess something up with validation? You have to blur the field for the submit button to disable or else it will be enabled

Reply:
Same behavior in my PR too so not sure what changed but from user perspective it's the same and I updated the test to be more reliable